docs/how-to-use-runhouse.rst

```diff
@@ -0,0 +1,285 @@
+from a blank page in your IDE.
+
+1. Define Compute
+^^^^^^^^^^^^^^^^^
+Runhouse allows you to define compute requirements in code, and launch ephemeral clusters from the compute pool we described in the prior section.
+Here, you can define the required CPU, GPU, memory, and disk requirements (or name a specific cluster) to use.
+For instance, to create a cluster on AWS with an A10 GPU attached using an Docker image, you can write:
+
+.. code:: python
+
+    import runhouse as rh
+
+    cluster = rh.ondemand_cluster(
+        name="rh-cluster", # This cluster can be saved and reused by name. We will prefix your username when saved, e.g. /my_username/rh-cluster
+        instance_type="A10G:1", # There are a number of options available for instance_type, check out the docs to see them all
+        provider="aws", # Specify a cloud provider
+        image_id="docker:nvcr.io/nvidia/pytorch:23.10-py3", # Use a Docker image
+        autostop_mins=90 # Remember to set autostop_mins to avoid leaving clusters running indefinitely.
+    ).up_if_not()
+
+You can run CLI commands on the cluster using ``cluster.run()`` to layer on setup steps beyond the underlying image; for instance, installing other packages.
+
+.. code:: python
+
+    cluster.run(['pip install numpy'])
+
+You can find full documentation about the Runhouse cluster API `in the Cluster docs <https://www.run.house/docs/tutorials/api-clusters>`_.
+
+1a. Starting the Runhouse Server Daemon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If not already running, the client will start the Runhouse API server daemon
+on the compute and form a secure network connection (either over SSH or HTTP/S).
+
+* The daemon can be thought of as a "Python object server," holding key-value pairs of names and Python objects in memory (objects you will dispatch
+to it in the next step), and exposing an HTTP API to call methods on those objects by name.
+* By default, objects are held in a single default worker process but can be sent to other worker processes, including on other nodes in the cluster,
+to achieve powerful parallelism out of the box.
+* When the object is used, and there is a ``GET http://myserver:32300/my_object/my_method``, the daemon will look up the object named "my_object,"
+issue an instruction for its worker to call the method "my_method" on it, and return the result.
+* The HTTP server and workers can handle thousands of concurrent calls per second, and have similar latency to Flask under most conditions.
+
+2. Dispatch Your Code
+^^^^^^^^^^^^^^^^^^^^^^
+Once you have established a connection to compute, the development pattern is to continuously dispatch code to the cluster and execute it there.
+You are doing local-like execution and debug, but with the power of the remote compute. Runhouse is agnostic to whether you dispatch
+using a Notebook or run directly from a Python script (dispatch should live within a ``if __name__ == "__main__":`` block in a script).
```

Comment:
maybe we add this as a "note" or some other callout? seems to have tripped up some users in the past